# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/security/test/general/test_cache_split.html
# Commit: 3dfd3c94a105
# Full Hash: 3dfd3c94a105e095aada0b356f1106370de222d3
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-12-07 15:51:17
# Regressor Bug: 1701828
# File Overlap Count: 1
# Description:
#   Bug 1701828 - meta charset rewrite. r=smaug
#   
#   Implements https://github.com/whatwg/html/issues/6962 . Improves performance
#   when <meta charset> occurs in head but after the first kilobyte and aligns
#   behavior better with WebKit and Blink.
# ==============================================================================

diff --git a/dom/security/test/general/test_cache_split.html b/dom/security/test/general/test_cache_split.html
--- a/dom/security/test/general/test_cache_split.html
+++ b/dom/security/test/general/test_cache_split.html
@@ -1,12 +1,13 @@
 <!DOCTYPE HTML>
 <html>
 
 <head>
+    <meta charset="utf-8">
     <title>Bug 1454721 - Add same-site cookie test for about:blank and about:srcdoc</title>
     <script src="/tests/SimpleTest/SimpleTest.js"></script>
     <script src="/tests/SimpleTest/ChromeTask.js"></script>
     <link rel="stylesheet" type="text/css" href="/tests/SimpleTest/test.css" />
 </head>
 
 <body>
     <img id="cookieImage">